tasks/certbot_nginx.yml

```diff
@@ -110,6 +110,11 @@
     path: "/etc/letsencrypt/live/{{ certbot_domain }}/fullchain.pem"
   register: cert_file
 
+- name: configure certbot
+  template:
+    src: 'certbot.ini.j2'
+    dest: "/etc/letsencrypt/cli.ini"
```

Comment:
Just wondering if /etc/letsencrypt directory is sure to already exist at this point? If unsure, this task could be placed underneath the `create certificates with certbot` task.

Reply:
As long as the certbot package is installed before, yes. The folder will be created along the way.

In case you feel that this might not be enough certainty, please respond to this comment (and I will create a new PR to make sure of its existance).